code/modules/hippie/items/stapler.dm

```diff
@@ -174,4 +174,75 @@
 			else
 				user << "<span class='notice'>There is already a butt in \the [src]!</span>"
 		else
-			user << "<span class='notice'>There is already a paper in \the [src]!</span>"
\ No newline at end of file
+			user << "<span class='notice'>There is already a paper in \the [src]!</span>"
+
+//coffin nailing for stapler instead of welding
+/obj/structure/closet/coffin/update_icon()
```

Comment:
eh, closets.dm handles this already? The only difference is `overlays += "nailed"`

https://github.com/HippieStation/HippieStation13/blob/42915e2f0fc726df31f1cebd7f3206907862c391/code/game/objects/structures/crates_lockers/closets.dm#L48